tests/installation/user_settings.pm

```diff
@@ -35,12 +35,19 @@ sub run {
     }
 
     $self->enter_userinfo();
-    assert_screen 'inst-userinfostyped';
-    if (match_has_tag 'boo#1122804') {
-        diag "trying again with slow typing due to https://progress.opensuse.org/issues/46190\n";
-        $self->enter_userinfo(retry => 1);
-        assert_screen 'inst-userinfostyped';
+    assert_screen 'inst-userinfostyped-ignore-full-name';
+    for (my $i = 0; $i < 3; $i++) {
+        if (check_screen('inst-userinfostyped-expected-typefaces', 0)) {
+            last;
+        }
+        else {
+            record_info('poo#46190', 'Typing issue with fullname - poo#46190');
+            diag "trying again with slow typing due to https://progress.opensuse.org/issues/46190\n";
```

Comment:
the '\n' is not needed, i'd use a record info instead, so that we know without lookin in the os-autoinst log, what happened.